v03_pipeline/lib/migration/misc_test.py

```diff
@@ -0,0 +1,44 @@
+import os
+import shutil
+import tempfile
+import unittest
+from unittest.mock import ANY
+
+from v03_pipeline.lib.migration.misc import list_migrations
+
+
+class TestListMigrations(unittest.TestCase):
+    def setUp(self):
+        self.tmpdir = tempfile.TemporaryDirectory()
+        for migration in [
+            '__init__.py',
+            '1111_a_migration.py',
+            '0000_migration.py',
+            '001_test.py',
+            'abcd_test.py',
+            '0000_migration.txt',
+        ]:
+            with open(os.path.join(self.tmpdir.name, migration), 'w') as f:
+                f.write(
+                    """
+from v03_pipeline.lib.migration.base_migration import BaseMigration
+class ImplementedMigration(BaseMigration):
+    pass
+                    """,
+                )
+
+    def tearDown(self):
+        if os.path.isdir(self.tmpdir.name):
+            shutil.rmtree(self.tmpdir.name)
+
+    def test_list_migrations(self):
+        self.assertEqual(
+            list_migrations(self.tmpdir.name),
+            [
+                ('0000_migration', ANY),
```

Comment:
what's the second item in the tuple returned by list_migrations? is it possible to assert on something more specific here

Reply:
Yeah, it's an implementation of the `BaseMigration` class.  I can make it more specific for sure!

Reply:
@jklugherz I have no idea if this was the best fix but it is fixed!